docs/src/rules/no-unused-vars.md

```diff
@@ -333,7 +333,7 @@ Examples of **correct** code for the `{ "caughtErrorsIgnorePattern": "^ignore" }
 ::: correct
 
 ```js
-/*eslint no-unused-vars: ["error", { "caughtErrorsIgnorePattern": "^ignore" }]*/
+/*eslint no-unused-vars: ["error", { "caughtErrors": "all", "caughtErrorsIgnorePattern": "^ignore" }]*/
```

Comment:
Good catch!

Reply:
While at it, let's also [update the docs here](https://github.com/eslint/eslint/blob/main/docs/src/rules/no-unused-vars.md#options) to mention the default for caughtErrors:

```diff
{
    "rules": {
-        "no-unused-vars": ["error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }]
+        "no-unused-vars": ["error", { "vars": "all", "args": "after-used", "caughtErrors": "none", "ignoreRestSiblings": false }]
    }
}
```